include/igl/eigs.cpp

```diff
@@ -33,8 +34,8 @@ IGL_INLINE bool igl::eigs(
   assert(iB.rows() == n && "B should be match A's dims.");
   assert(iB.cols() == n && "B should be square.");
   assert(type == EIGS_TYPE_SM && "Only low frequencies are supported");
-  DerivedU U(n,k);
-  DerivedS S(k,1);
+  DerivedU U(n, 1);
```

Comment:
I don't understand this change. Want this lead to errors below when calling `U.col(i)`?